web/src/pages/guidance/published.module.scss

```diff
@@ -0,0 +1,14 @@
+@use '../../../node_modules/@nice-digital/nds-core/scss/core';
```

Comment:
It's a shame we need the relative path here. Can we not use a tilde or possibly better, use:

```
sassOptions: {
    includePaths: [path.join(__dirname, 'node_modules/@nice-digital')],
  },
```

so we can do:

```
@use 'nds-core/scss/core'
```

e.g. https://github.com/vercel/next.js/pull/11063/files#diff-88cfae57d8e095126854cbe4fe5d714bedd8a06f264c5a481b5df57ccc20733dR6

Probably makes sense as we'll end using nds-core in lots of places for the vars/funcs/mixins so a shorter, absolute path is probably better.